### ISSUE TITLE
Axis of Gradient Node Vectors

### ISSUE THREAD

**Issue by kevinmkarol** [spritebuilder/SpriteBuilder#698]
A positive x vector results in a gradient that goes along the negative x axis, and a positive y vector results in a gradient that goes along the negative y axis, not following standard axis conventions.

**Comment by NickyWeber** [comment]
Where can I find the standard axis conventions? :)

**Comment by kevinmkarol** [comment]
http://en.wikipedia.org/wiki/Cartesian_coordinate_system
http://en.wikipedia.org/wiki/Euclidean_vector
Currently a positive x,y vector points the transition from start color to end color in the direction of the 3rd quadrant, which is normally a negative x,y vector

**Comment by slembcke** [comment]
Yeah... The problem is that it works that way in Cocos2D and has for some time apparently. I first noticed it when rewriting the rendering code for it for 3.1. I had "fixed" it accidentally and the rendering tests weren't working the same as 3.0 anymore. For a minor version, I figured it was more important to be consistent since it wasn't a serious bug.

**Comment by NickyWeber** [comment]
Then let's put it on the roadmap for v4 and don't change the winning team.
